File: Geeks4geeks/Mathworks/Pair/amex.cpp
#include <iostream>
using namespace std;

int gcd(int a, int b) 
{ 
    if (b == 0) 
        return a; 
    return gcd(b, a % b);  
      
} 
 
// Calculating the Euler Totient Function

int Totient( int n ){
    int result = 1;
    for ( int i=2 ; i<=n ; i++ ){
        if ( gcd(i,n) == 1 ){
            result+=1;
        }
    }
    return result;
}

// Driver code:

int main(){
    int T;
    cin >> T;
    while (T--){
        int N;
        cin >> N;
        for ( int i=1 ; i<N+1 ; i++ ){
            cout << Totient(i) << " ";
        }
        cout <<endl;
    }
}

File: Geeks4geeks/Mathworks/Pair/amex.py
def gcd(a, b): 
    if (a == 0): 
        return b 
    return gcd(b % a, a) 


# A simple method to evaluate 
# Euler Totient Function 

def phi(n):
    result = 1
    for i in range(2, n): 
        if (gcd(i, n) == 1): 
            result+=1
    return result 


# Driver Code 
for _ in range(int(input())):
  k = int(input())
  for n in range(1, k+1): 
    print(phi(n),end=" ")
  print()